src/PointCloud/Core/PointCloudOctant.cs

```diff
@@ -193,6 +194,56 @@ public bool Intersects(double3 point)
             (point.z >= Min.z && point.z <= Max.z);
         }
 
+
+        /// <summary>
+        /// Returns true if the shape is completely inside or is intersecting this <see cref="PointCloudOctant"/>
+        /// </summary>
+        /// <param name="invSphereModel">The inverse model matrix which is currently used to render the <see cref="Sphere"/> (can e. g. be displayed as an ellipsoid)</param>
+        /// <returns></returns>
```

Comment:
Maybe:

///Returns true if an ellipsoid is completely inside or is intersecting this <see cref="PointCloudOctant"/>.
///<param name="invSphereModel">The inverse transformation matrix that transforms an ellipsoid into a sphere.</param>